### DISCUSSION TITLE
Get signed APK from Play Store

### DISCUSSION THREAD

**Discussion by qpre** [fastlane/fastlane#20725] [Q&A]
Hello,

Now that signing is deferred to the Play Store, I was wondering if there was an easy (as in "from the CI") way to download the generated APK from the Play Store without going through the App Bundle Explorer.

Thanks !

**Comment by zhgchgli0718** [comment]
I wrote a simple extension to the Fastlane Action that calls the Google API to download the uploaded APK. The code is available here: https://gist.github.com/zhgchgli0718/42f6326225e393e20616ce15f3390c88.
